src/index.ts

```diff
@@ -0,0 +1,6 @@
+import app from './app';
+import config from './config';
+
+app.listen(config.port, () => {
+  console.log(`I am listening at http://localhost:${config.port}`);
+});
```

Comment:
I put this in its own file so that app can be exported from app.ts for testing purposes without leaving the server open